vts/appraisal/appraisal.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package appraisal
+
+import (
+	"github.com/veraison/ear"
+	"github.com/veraison/services/proto"
+)
+
+type Appraisal struct {
```

Comment:
I am struggling a bit to understand the use of Appraisal in context of:
May be its just me, but could you please add some comments to clarify how Appraisal is used in the flow ?
Especially given the fact that the interface is as below

func (s *RPCClient) AppraiseEvidence(ec *proto.EvidenceContext, endorsements []string) (*ear.AttestationResult, error) {

Reply:
It is the equivalent of `proto.AppraisalContext` but not tied to protobufs. It ties together the evidence and the attestation result. The difference is that it includes the actual attestation result and so can be used internally by VTS (specifically, by the policy manager). Since we're now using the EAR attestation result this is not defined as a protobuf message`, `proto.AppraisalContext` must now have it in a serialized form, which means it's no longer usable inside VTS, and is purely the definition of the protobuf message.

As requested, I've added a comment to the struct defintion.